output.css

```diff
@@ -26784,6 +26784,14 @@ video {
   visibility: hidden;
 }
 
+.d-none {
+  display: none;
+}
+
+.d-grid {
+  display:grid;
+}
+
```

Comment:
I'm going to need you to run `yarn build:css`. This file was never meant to be manually edited. It is simply the result of a build process.

The file you would edit (but you don't need to because I'm using TailwindCSS) would be `styles.css`.

If you haven't already you'll need to run `yarn` first:

```bash
# installs dependencies
yarn
# Builds CSS
yarn build:css
```

**Make sure you do this after you're done changing the HTML first!**